import-service/lib/import-service-stack.ts

```diff
@@ -76,6 +81,20 @@ export class ImportServiceStack extends cdk.Stack {
             }
         );
 
+        const basicAuthorizerLambda = lambda.Function.fromFunctionArn(
+            this,
+            "BasicAuthorizer",
+            process.env.BASIC_AUTH_LAMBDA_ARN as string
```

Comment:
how do you get this ARN? Should it be set manually after separate auth stack deployment?

May be not the best way. You can get lambda function reference with "fromFunctionName".
Or if you want to use "fromFunctionArn", you can pass this arn as a prop to import stack (this approach will work if you app is created as an app with multiple stacks)